packages/components/src/components/primitives/Button/Button.tsx

```diff
@@ -1,5 +1,20 @@
 import React, { FC } from 'react';
-import { Button as ChakraButton } from '@chakra-ui/react';
+import { Button as ChakraButton, useMultiStyleConfig } from '@chakra-ui/react';
 import { ButtonProps } from './types';
+import { Spinner } from '..';
+import { SpinnerProps } from '../Spinner/types';
```

Comment:
please re-export the `SpinnerProps` from `components/primitives/Spinner` instead, we missed that for some components

Reply:
done!